### ISSUE TITLE
[OTHER]: Links are not clickable in caption and post

### ISSUE THREAD

**Issue by Kalyan-velu** [narayan954/dummygram#655]
### What would you like to share?

In the past, it was possible to click on links embedded within post content and captions. Unfortunately, it appears that this feature has been unintentionally removed. If not removed intentionally, I would like to address this issue and work on reinstating the ability to click on links again.

**Comment by narayan954** [comment]
yes exactly, glad you remembered, I was also going to raise the issue, can you add it back from pr #486 and test it again?
thanks
